bazel/verilog.bzl

```diff
@@ -142,11 +142,15 @@ def _verilog_base_impl(ctx, subcmd, test = True, extra_args = [], extra_runfiles
         )
 
         # Run generator script
+
         ctx.actions.run(
```

Comment:
Ah, maybe the fix is actually `ctx.actions.run_shell`, that might inherit the env automatically

Reply:
I tried and it doesn't work, it still requires passing the environment variable explicitly

Reply:
https://chatgpt.com/share/673fa301-bf20-8006-bf02-158121650609 says:

> bazel --action_env not used during build ctx.actions.run
ChatGPT said:
When using Bazel's ctx.actions.run to create build actions, environment variables specified with the --action_env flag are not automatically applied. This behavior is due to the default environment isolation in Bazel's build actions, which ensures hermetic builds by not inheriting environment variables from the host system.
> 
> To include specific environment variables in your build actions, you have two options:
> 
> Use use_default_shell_env=True: This option allows the action to inherit the default shell environment, including variables specified with --action_env. However, be aware that setting use_default_shell_env=True can override any environment variables you specify directly in the env parameter of ctx.actions.run. This means that if both use_default_shell_env=True and env are set, the environment variables from env may not be applied as intended. 
[GitHub](https://github.com/bazelbuild/bazel/issues/5980?utm_source=chatgpt.com)
> 
> Manually specify environment variables: You can explicitly define the environment variables for the action by using the env parameter in ctx.actions.run. This approach provides precise control over the environment but requires you to list all necessary variables manually.
> 
> It's important to note that Bazel's handling of environment variables in build actions is designed to promote build reproducibility and isolation. By default, build actions do not inherit environment variables from the host system to prevent unintended side effects. Therefore, when configuring environment variables for build actions, it's crucial to be explicit and deliberate to maintain the integrity of your builds.
> 
> For more detailed information on this topic, you can refer to discussions in the Bazel community, such as the one on the Bazel GitHub repository.

Try `use_default_shell_env=True`?

Reply:
done, it works!